### ISSUE TITLE
[Neo4j Browser; Cypher] No captions on some nodes after LOAD CSV, MERGE ...

### ISSUE THREAD

**Issue by victoriastuart** [neo4j/neo4j#11461]
- Neo4j version: 3.3.4
- Operating system: Arch Linux x86_64  |  XFCE
- API/Driver: Cypher { cypher-shell: .../neo4j-community-3.3.4/bin/neo4j-shell | cycli 0.7.6}
- Web browser: Firefox 59.0.2 (64-bit)
 -- ditto: Chrome 65.0.3325.181 (Official Build) (64-bit)  |  Opera 52.0.2871.40 
- Neo4j Browser version: 3.1.4

See attached screenshots.

I'm not sure if this is a bug (likely not; if that is the case, then perhaps consider this Issue as a Feature Request?).

### Steps to reproduce

* Cypher script (`glycolysis3z.cypher`):

```
LOAD CSV WITH HEADERS FROM "file:/mnt/Vancouver/Programming/data/metabolism/pg2neo4j/glycolysis_relations2.csv" AS row
MERGE (s:GlycolysisRelations {source:row.source})
MERGE (t:GlycolysisRelations {target:row.target})
  RETURN labels(s), labels(t), s.source, t.target;
```

* CSV data file (`glycolysis_relations2.csv`):

```
source,relation,target
hexokinase,yields,glucose 6-phosphate
glucose 6-phosphate,substrate_of,glucose-6-phosphatase
```

* Cycli output:

```
(py27) [victoria@victoria]$ cycli -P 7474 -u victoria -p <password_obfuscated_here> --filename /mnt/Vancouver/Programming/data/metabolism/glycolysis3z.cypher

> LOAD CSV WITH HEADERS FROM "file:/mnt/Vancouver/Programming/data/metabolism/pg2neo4j/glycolysis_relations2.csv" AS row
MERGE (s:GlycolysisRelations {source:row.source})
MERGE (t:GlycolysisRelations {target:row.target})
  RETURN labels(s), labels(t), s.source, t.target;

labels(s)                | labels(t)                | s.source            | t.target             
-------------------------+--------------------------+---------------------+----------------------
[u'GlycolysisRelations'] | [u'GlycolysisRelations'] | hexokinase          | glucose 6-phosphate  
[u'GlycolysisRelations'] | [u'GlycolysisRelations'] | glucose 6-phosphate | glucose-6-phosphatase

108 ms

(py27) [victoria@victoria]$
```

* cypher-shell output:
```

[victoria@victoria]$ cat glycolysis3z.cypher |  cypher-shell -u victoria -p password_obfuscated_here
labels(s), labels(t), s.source, t.target

["GlycolysisRelations"], ["GlycolysisRelations"], "hexokinase", "glucose 6-phosphate"
["GlycolysisRelations"], ["GlycolysisRelations"], "glucose 6-phosphate", "glucose-6-phosphatase"

[victoria@victoria]$ 
```

### Expected behavior

That all nodes be properly captioned, in Neo4j Browser, per the specifications in the Cypher script and the fields in the CSV. 

E.g., in the attached screenshot, that all `source` nodes be captioned with the appropriate source name (from the CSV), **and also** that all the `target` nodes be simultaneously labeled with the appropriate target names (again, from the CSV).

### Actual behavior

* Missing captions on any node that is not at index position [0] in the data file.  See comments above + screenshots.

[ *I had to upload the .cypher script and .csv files with a .txt suffix; drop the `.txt` portion to get the actual (cited) files.* ] 

[glycolysis_relations2.csv.txt](https://github.com/neo4j/neo4j/files/1876928/glycolysis_relations2.csv.txt)

[glycolysis3z.cypher.txt](https://github.com/neo4j/neo4j/files/1876930/glycolysis3z.cypher.txt)

![selection_102](https://user-images.githubusercontent.com/2575920/38324635-ba910326-37f5-11e8-947c-6ac80928c9c5.png)

![selection_103](https://user-images.githubusercontent.com/2575920/38324634-ba79990c-37f5-11e8-8f93-c13638a32918.png)

![selection_104](https://user-images.githubusercontent.com/2575920/38324633-ba606e78-37f5-11e8-85ad-a64f47d76e72.png)

**Comment by oskarhane** [comment]
Hi,

Thanks for the report.
If I understand correctly, I think this is expected behaviour. Nodes of a certain label can only show one caption at a time.
Two of your nodes with the label `GlycolysisRelations` have the `source` property set and the other two have the `target` property set. These cannot be show at the same time.

I also think the model looks a bit odd, source and target should probably be nodes and have the relation between them.

I'd suggest you jump into our public Slack community: http://neo4j.com/slack

**Comment by victoriastuart** [comment]
@oskarhane: thank you; much appreciated ... I thought as much (a modeling issue).

I created a Slack account earlier today, but the free account has limited space (an uploaded PNG file exceeded storage quota), and I find that the chat forum format is not conducive to a complex QA discussion, in my opinion.  It looks like a great forum for quick QA, though!  ;-)

**Update / Solution:**

I posted this "issue" to: https://stackoverflow.com/questions/49682338/creating-a-metabolic-pathway-in-neo4j, where I received an excellent answer.  Thanks again to all who took the time to address this request  :-D

---

```
LOAD CSV WITH HEADERS FROM "file:/glycolysis_relations.csv" AS row
MERGE (s:Glycolysis {name: row.source})
MERGE (t:Glycolysis {name: row.target})
WITH s, t, row
CALL apoc.create.relationship(s, row.relation, {}, t) YIELD rel
RETURN COUNT(*);
```

![neo4j_browser-glycolysis 2018 04 05b](https://user-images.githubusercontent.com/2575920/38401479-d5f3fc5c-390a-11e8-8069-05b394af30fa.png)
